src/window.c

```diff
@@ -51,7 +51,13 @@ LIBUI_FUNCTION(create) {
 	ARG_STRING(title, 0);
 	ARG_INT32(width, 1);
 	ARG_INT32(height, 2);
-	ARG_BOOL(has_menubar, 3);
+
```

Comment:
Other coercions are normally done on JS classes:

https://github.com/mischnic/libui-napi/blob/568297e4478571d92f6678b73e294910ce404c32/js/window.js#L103-L105

We should do this there too